demonstrations/tutorial_how_to_build_compressed_double_factorized_hamiltonians.py

```diff
@@ -0,0 +1,382 @@
+r"""How to build compressed double-factorized Hamiltonians
+==========================================================
+
+Compressed double factorization offers a powerful approach to overcome key limitations in
+quantum chemistry simulations. Specifically, it tackles the runtime's dependency on the
+Hamiltonian's one-norm and the shot requirements linked to the number of terms [#cdf]_. In this
+tutorial, we will learn how to construct the electronic Hamiltonian in the compressed
+double-factorized form using tensor contractions. We will also show how this technique allows
+having a linear combination of unitaries (LCU) representation suitable for error-corrected
+error-corrected algorithms, which facilitates efficient simulations via linear-depth circuits
+with Givens rotations.
+
+Constructing the electronic Hamiltonian
```

Comment:
Recently, we've been adding the image to the intro paragraph for a nice visual.

```suggestion
Compressed double factorization offers a powerful approach to overcome key limitations in
quantum chemistry simulations. Specifically, it tackles the runtime's dependency on the
Hamiltonian's one-norm and the shot requirements linked to the number of terms [#cdf]_. In this
tutorial, we will learn how to construct the electronic Hamiltonian in the compressed
double-factorized form using tensor contractions. We will also show how this technique allows
having a linear combination of unitaries (LCU) representation suitable for error-corrected
error-corrected algorithms, which facilitates efficient simulations via linear-depth circuits
with Givens rotations.

.. figure:: ../_static/demo_thumbnails/opengraph_demo_thumbnails/OGthumbnail_how_to_build_cdf_hamiltonians.png
    :align: center
    :width: 70%
    :target: javascript:void(0)

Constructing the electronic Hamiltonian
```

Reply:
added in commit: https://github.com/PennyLaneAI/qml/pull/1319/commits/25d5bf9b0bfad53b396ed86b7ddcb27c4b66df55